pom.xml

```diff
@@ -53,6 +54,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.rootdir>${project.basedir}</project.rootdir>
     <scala.binary.version>2.12</scala.binary.version>
```

Comment:
Why do `scala.binary.version` and `dep.cask.version` have "default" here, which gets overwritten by the profile, while `scala.version` does not have such a default? Is there a specific reason? 

If not, I guess, I would prefer to either have defaults for all profile-related properties OR for none of them.

Reply:
The [docs for the cross-build-plugin](https://github.com/kelnos/scala-cross-maven-plugin#usage) suggest it's a good idea to keep scala.binary.version to keep IntelliJ happy. I suspect the other is an oversight, I will try and remove it.